src/components/Questionnaire/QuestionnaireResults/Table/TableILS.tsx

```diff
@@ -0,0 +1,179 @@
+import { ILS } from '@core'
+
+//Returns the Interpretation of the ILS-Test (balanced, moderate, strong + Dimension (if score is not balanced))
+export const ILSInterpretation = (score: number, interpretationString: string, onlyEnglish?: boolean): string => {
+  const { t, i18n } = useTranslation()
+
+  const inter = new Map<number, string>()
+
+  if (onlyEnglish) {
+    const en = i18n.getFixedT('en')
+
+    inter.set(-1, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(1, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-3, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(3, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-5, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(5, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-7, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(7, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-9, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(9, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(-11, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(11, en('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+  } else {
+    inter.set(-1, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(1, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-3, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(3, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
+    inter.set(-5, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(5, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-7, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(7, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.moderate'))
+    inter.set(-9, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(9, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(-11, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+    inter.set(11, t('components.Questionnaire.QuestionnaireResults.Table.TableILS.strong'))
+  }
+
+  //if the interpretation is "balanced", then only return "balanced" without the Dimension
+  if (inter.get(score) === t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced'))
```

Comment:
Option1 ? Inter.get(score) : option2

Statt der if

Reply:
gibt nur if und keine else

Reply:
das zweite return ist basically das else. is aber nur notation, beides funktioniert gleich.

```suggestion
  return inter.get(score) === t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced') ? t('components.Questionnaire.QuestionnaireResults.Table.TableILS.balanced') : inter.get(score) + ' ' + interpretationString
```